packages/template-retail-react-app/app/utils/jwt-utils.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {createRemoteJWKSet as joseCreateRemoteJWKSet, jwtVerify} from 'jose'
+import {getAppOrigin} from '@salesforce/pwa-kit-react-sdk/utils/url'
+import {getConfig} from '@salesforce/pwa-kit-runtime/utils/ssr-config'
+
+const throwSlasTokenValidationError = (message, code) => {
+    throw new Error(`SLAS Token Validation Error: ${message}`, code)
+}
+
+export const createRemoteJWKSet = () => {
+    const appOrigin = getAppOrigin()
+    const {app: appConfig} = getConfig()
+    const shortCode = appConfig.commerceAPI.parameters.shortCode
+    const tenantId = appConfig.commerceAPI.parameters.organizationId.replace(/^f_ecom_/, '')
```

Comment:
The code assumes the values `shortCode` and `tenantId` from the static PWA Kit config instead of extracting and validating them directly from the SLAS token.

Probably the assumption is correct in most cases, but it’s not the safest approach. My understanding is that tokens should be validated based on the data they carry to ensure they match their intended context.

Reply:
Would you consider doing both things? i.e.
1. Extract the `shortCode` and `tenantId` from the token.
2. Validate the values match the PWA Kit config values.